ca/ca_test.go

```diff
@@ -356,7 +356,7 @@ func TestIssueCertificate(t *testing.T) {
 
 				issueReq := &caPB.IssueCertificateRequest{Csr: testCase.csr, RegistrationID: &arbitraryRegID}
 
-				certDER := []byte{}
+				var certDER []byte
```

Comment:
Was this flagged by ineffassign?

Reply:
Yup, because the assigned value is never used, just overwritten.

Reply:
(`var certDER []byte` == `nil`, whereas `[]byte{}` is an actual slice)